### PR TITLE
Adding Symfony4 support #SymfonyConHackday2017

### PR DESCRIPTION
**Pull request by Legenyes**

### CODE REVIEW

**Comment by Legenyes** [comment]
What Can i do ?

**Comment by craue** [comment]
[See above](https://github.com/craue/TwigExtensionsBundle/pull/12#pullrequestreview-78354166), @Legenyes. 😏

File: .travis.yml [hunk 1]
```diff
@@ -29,11 +29,16 @@ matrix:
...
+    - php: 7.1
+      env: SYMFONY_VERSION='4.0.*' TWIG_VERSION='~2.0'
   allow_failures:
-    - env: SYMFONY_VERSION='3.4.*' TWIG_VERSION='~2.0' MIN_STABILITY='dev'
+    - env: SYMFONY_VERSION='3.4.*' TWIG_VERSION='~2.0'
   fast_finish: true
 
+before_install:
+    - if [ "$DEPENDENCIES" = "beta"  ]; then composer config minimum-stability beta; fi;
+
```
Comment (craue) [inline .travis.yml]: This is not needed and can be removed.

File: Resources/config/twig/FormExtension.xml [hunk 2]
```diff
@@ -7,14 +7,14 @@
 <container
 		xmlns="http://symfony.com/schema/dic/services"
 		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+		xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd" >
```
Comment (craue) [inline Resources/config/twig/FormExtension.xml]: The blank before `>` should be removed.

**Comment by craue** [comment]
Again, what is this "recipe" for?

**Comment by Legenyes** [comment]
for recipe, here is the documentation
https://symfony.com/doc/current/setup/flex.html

> If there's special information about that package, Flex returns it in a file called a "recipe" and the application uses it to decide which package to install and which automated tasks to run after the installation.

**Comment by craue** [comment]
I'm not sure adding such a recipe here is justified.

> [Don't create a recipe for Symfony bundles if the only configuration in the manifest is the registration of the bundle for all environments, as this is done automatically.](https://github.com/symfony/recipes/blob/master/README.rst#creating-recipes)

What would be this "special information"? Apart from that...

> [Recipes must be stored on their own repositories, outside of your Composer package repository.](https://github.com/symfony/recipes/blob/master/README.rst#creating-recipes)

If, at all, it should go into a separate PR as it's not necessarily needed to make the bundle Symfony 4 compatible.

**Comment by Legenyes** [comment]
Your bundle require this adding:

> Make the Twig extensions available by updating your configuration
> 
> # in app/config/config.yml
> craue_twig_extensions: ~

The recipe is for that. I can remove the recipe if your bundle run without adding a config line in the config.yml

**Comment by craue** [comment]
I see, but please open a new PR for that. It shouldn't be part of this one.

**Comment by craue** [comment]
Thank you, @Legenyes.

**Pull request merged**